Gems/ROS2/Code/Source/VehicleDynamics/DriveModels/SkidSteeringDriveModel.h

```diff
@@ -37,25 +37,43 @@ namespace ROS2::VehicleDynamics
         AZStd::pair<AZ::Vector3, AZ::Vector3> GetVelocityFromModel() override;
 
     private:
+        // Data collected once and cached for the reuse
```

Comment:
This comment should be next to the member (m_wheelsCache instead). 
Instead, describe what the structure is for here.